Library/Sender.ts

```diff
@@ -59,6 +64,9 @@ class Sender {
         this._fileCleanupTimer = null;
         // tmpdir is /tmp for *nix and USERDIR/AppData/Local/Temp for Windows
         this._tempDir = path.join(os.tmpdir(), Sender.TEMPDIR_PREFIX + this._config.instrumentationKey);
+        this._isStatsbeatSender = isStatsbeatSender || false;
+        this._shutdownStatsbeat = shutdownStatsbeat;
+        this._failedToIngestCounter = 0;
```

Comment:
line 36 is default to 0, right? line 69 might be redundant? at least that's what Java does.

Reply:
Default value would be undefined in this case even if we set the type as number